bundles/SyliusReviewBundle/container-configuration.rst

```diff
@@ -0,0 +1,39 @@
+Container configuration
+=======================
+
+Using *SyliusReviewBundle*, you should configure your review resource with several key. They'll be automatically processed
+to proper configuration and database mapping. Whole configuration should be written under ``sylius_review`` tag in your ``app/config/config.yml`` file.
+Under ``sylius_review``, ``classes`` key is required, which defines tree of specific reviews configuration. Next tag should be named according
+to your "reviewable" entity name (see exemplary configuration at the end of this page).
+
+Configuration details
+---------------------
+
+Keys that should be configured:
+
+- ``subject`` (required): entity representing reviewable resource (remember it must implements ``Sylius\Component\Review\Model\ReviewableInterface``)
+- ``review``: tree that describes your review class; it should contains following sub-keys:
+    - ``model`` (required): your review model; it must implements ``Sylius\Component\Review\Model\ReviewInterface``
+    - ``controller``: custom controller for your review class, as default it is ``Sylius\Bundle\ResourceBundle\Controller\ResourceController``
```

Comment:
Or *The default value is `Sylius\Bundle\ResourceBundle\Controller\ResourceController`*